.circleci/config.yml

```diff
@@ -628,6 +628,7 @@ workflows:
   check_buildtype_device:
     jobs:
       - detekt
+      - firebase_screenshots
```

Comment:
on purrpose?

Reply:
yes, so that reviewers can easily verify it without the need of running them by themselves